api/index.ts

```diff
@@ -8,17 +8,26 @@ const dataDir: string = process.env.DATA_DIR || "/config";
 const app = express();
 app.use(cors());
 
-app.get("/:device/:mode?", (req: express.Request, res: express.Response) => {
+app.get("/:device/:remote/:mode?", (req: express.Request, res: express.Response) => {
```

Comment:
Watch out, the `:variable` syntax prefixed with `:` means that's a dynamic variable